drive/modules/listener/watch_schedular.bal

```diff
@@ -118,7 +134,7 @@ class Job {
 
     isolated function scheduleNextChannelRenewal() returns error? {
         time:Utc currentUtc = time:utcNow();
-        decimal timeDifference = (self.expiration/1000) - (<decimal>currentUtc[0]) - 60;
+        decimal timeDifference = (self.expiration/1000) - (<decimal>currentUtc[0]) - <decimal>self.leadTime;
         time:Utc newTime = time:utcAddSeconds(currentUtc, timeDifference);
         time:Civil time = time:utcToCivil(newTime);
```

Comment:
If you want to just a log print of time you have used, you may use UTC to String function of the time module. then it will print it as in the YYYY:MM:DDT:HH:MM:SS:Z format. then we can remove time: Civil variable as it is a debug log print. 
``` 
// Converts a given `time:Utc` time to a RFC 3339 timestamp.
string utcString = time:utcToString(utc); 
```

Reply:
Ah great.. Thanks. Let's do that way